### PR TITLE
fix: handle promise rejection happening with cli-executor

### LINKED ISSUE #1268: Handle the promise rejection if an interrupt occurs in the regular workflow of cli-executor
**Describe the bug**

The `cli-executor` makes sure that the users aren't left out if they pass in an unknown flag. And if the user interrupts the regular workflow, a promise rejection occurs which is left unhandled.

**What is the current behavior?**

`[webpack-cli] Promise rejection:` is logged on to the console.

**To Reproduce**

Type in `webpack-cli --unknown`

`ctrl/cmd+c` would result in a promise rejection text being logged to the console.

**Expected behavior**

Handle the promise rejection thereby replacing what is currently logged with an appropriate message.

**Screenshots**

Screenshot from 2020-02-26 10-19-06

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
bugfix

**Did you add tests for your changes?**
Nope

**If relevant, did you update the documentation?**
N/A

**Summary**
closes #1268

**Does this PR introduce a breaking change?**\
Nope

**Other information**
N/A

### CODE REVIEW

**Comment by jamesgeorge007** [comment]
> Maybe we can have a more friendly error message

@ematipico got any suggestions?

**Comment by ematipico** [comment]
I don't at the moment. But you can come up with a better wording :)

File: packages/webpack-cli/lib/utils/cli-executer.js [hunk 1]
```diff
@@ -50,10 +50,14 @@ async function prompter() {
...
-    process.stdout.write('\n');
-    logger.info(`Executing CLI\n`);
-    runner([], args);
+    try {
+      const args = await prompter();
+      process.stdout.write('\n');
+      logger.info(`Executing CLI\n`);
+      runner([], args);	
+    } catch (err) {
+      logger.error('Interrupted')
```
Comment (anshumanv) [inline packages/webpack-cli/lib/utils/cli-executer.js]:
```suggestion
      logger.error('Action Interrupted, use `webpack-cli help` to see possible options.')
```

Maybe this? What say @ematipico

Comment (ematipico) [reply packages/webpack-cli/lib/utils/cli-executer.js]: Much better! :)

Comment (anshumanv) [reply packages/webpack-cli/lib/utils/cli-executer.js]: We can wrap `webpack-cli help` in chalk to colorize it maybe, that will be even better.

File: packages/webpack-cli/lib/utils/cli-executer.js [hunk 2]
```diff
@@ -50,10 +51,14 @@ async function prompter() {
...
-    process.stdout.write('\n');
-    logger.info(`Executing CLI\n`);
-    runner([], args);
+    try {
+      const args = await prompter();
+      process.stdout.write('\n');
+      logger.info(`Executing CLI\n`);
+      runner([], args);	
+    } catch (err) {
+      logger.error(`Action Interrupted, use ${chalk.cyan(`webpack-cli help`)} to see possible options.`)
```
Comment (snitin315) [inline packages/webpack-cli/lib/utils/cli-executer.js]: How about Action Interrupted in red and the rest in yellow? Will it be good?

**Pull request merged**